client/my-sites/importer/site-importer/site-importer.jsx

```diff
@@ -0,0 +1,92 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+
+import { appStates } from 'state/imports/constants';
+import Card from 'components/card';
+import ImporterHeader from '../importer-header';
+import ImportingPane from '../importing-pane';
+import SiteImporterInputPane from './site-importer-input-pane';
+
+/**
+ * Module variables
+ */
+const compactStates = [ appStates.DISABLED, appStates.INACTIVE ],
+	importingStates = [
```

Comment:
nit: should we use const for each line here?

Reply:
Updated.